meeteval/wer/wer/siso.py

```diff
@@ -42,12 +45,31 @@ def siso_word_error_rate(
     >>> siso_word_error_rate(reference='This is wikipedia', hypothesis='This wikipedia')  # Deletion example from https://en.wikipedia.org/wiki/Word_error_rate
     ErrorRate(errors=1, length=3, insertions=0, deletions=1, substitutions=0, error_rate=0.3333333333333333)
     """
+    if isinstance(reference, (KeyedText, STM)) or isinstance(hypothesis, (KeyedText, STM)):
```

Comment:
It is strange that this function accepts STM and KeyedText.
Could you give a hint, where this is helpful?

Reply:
I'm not sure if this function should accept STM. The command line interface doesn't accept STM. So, I might remove STM

Reply:
Yes, from the command line, it is clear, what to do with an STM:
 - SISO-WER on segments, like it is done in CHiME-5
 - SISO-WER on concatenated transcripts (cpWER without perm solver).
 - Assert that the above two yield the same WER, i.e. for each file only one transcript.